tileserver/__init__.py

```diff
@@ -128,6 +169,30 @@ def handle_request(self, request):
         coord = request_data.coord
         format = request_data.format
 
+        if self.store and layer_data is not self.layer_config.all_layers:
```

Comment:
Is this equivalent to the [store conditional](https://github.com/mapzen/tileserver/blob/use-store-dynamic-layers/tileserver/__init__.py#L211) lower down? If so, would it be better to use `layer_spec == 'all' and coord.zoom <= 20` for consistency, or assign that to a boolean so that we know we're only ever going to try and load/store the same set of tiles?